src/styles/leaderboard.css

```diff
@@ -302,4 +313,4 @@
   .leaderboard-table td:nth-of-type(8):before {
     content: "GAS";
   }
-}
+}
```

Comment:
In general, ending with a newline

```suggestion
}

```